src/settings/Settings.ts

```diff
@@ -817,4 +817,12 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         supportedLevels: LEVELS_UI_FEATURE,
         default: true,
     },
+    [UIFeature.ShowOneToOneRTCCall]: {
+        supportedLevels: LEVELS_UI_FEATURE,
+        default: false,
```

Comment:
Please set the defaults to the element base repo behaviour, which I believe would be true instead of false

Reply:
Done